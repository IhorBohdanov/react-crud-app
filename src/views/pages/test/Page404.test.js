import React from "react";
import { render, screen } from "@testing-library/react";
import Page404 from "../Page404";


describe("404 page", () => {

  it('should match snapshot', () => {
    const { asFragment } = render(<Page404 />);
    expect(asFragment(<Page404 />)).toMatchSnapshot();
  });
});