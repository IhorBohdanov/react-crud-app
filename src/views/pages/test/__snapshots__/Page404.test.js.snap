// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`404 page match snapshot 1`] = `
<DocumentFragment>
  <p>
    404
  </p>
</DocumentFragment>
`;
